cmd/login.go

```diff
@@ -11,11 +11,17 @@ func getCmdLogin(gs *state.GlobalState) *cobra.Command {
 	loginCmd := &cobra.Command{
 		Use:   "login",
 		Short: "Authenticate with a service",
-		Long: `Authenticate with a service.
+		Long: `[deprecation notice]
```

Comment:
As you're already using the `Deprecated` field, I think this change isn't longer needed. Otherwise, it looks a bit redundant:

```
❯ go run main.go login -h
Command "login" is deprecated, This command is deprecated and will be removed in a future release.
Please use the "k6 cloud login" command instead.

[deprecation notice]
This command is deprecated and will be removed in a future release. Please use the "k6 cloud login" command instead.

Authenticate with a service.
```

Reply:
Forgot this one behind 👍